pkg/api-server/global_insights_endpoints.go

```diff
@@ -1,42 +1,52 @@
 package api_server
 
 import (
+	"strings"
 	"time"
 
 	"github.com/emicklei/go-restful"
 
 	"github.com/kumahq/kuma/pkg/core"
 	"github.com/kumahq/kuma/pkg/core/resources/access"
-	"github.com/kumahq/kuma/pkg/core/resources/apis/mesh"
 	"github.com/kumahq/kuma/pkg/core/resources/apis/system"
 	"github.com/kumahq/kuma/pkg/core/resources/manager"
+	"github.com/kumahq/kuma/pkg/core/resources/model"
+	"github.com/kumahq/kuma/pkg/core/resources/registry"
 	rest_errors "github.com/kumahq/kuma/pkg/core/rest/errors"
 )
 
 type globalInsightsEndpoints struct {
 	resManager     manager.ResourceManager
 	resourceAccess access.ResourceAccess
+	resources      map[string]globalInsightsStat
```

Comment:
why do we need this variable to be a part of globalInsightsEndpoints? Shouldn't this be local to `inspectGlobalResources`?

Reply:
that's completely valid comment, I don't know why I did that to be honest :-D

Reply:
addressing now

Reply:
done